ludwig/benchmarking/utils.py

```diff
@@ -155,12 +159,57 @@ async def download_one(
     return dataset_name, local_dir
 
 
+def propagate_global_parameters(benchmarking_config: Dict[str, Any]):
+    """Propagate the global parameters of the benchmarking config to local experiments.
+
+    Args:
+        benchmarking_config: benchmarking config dictionary.
+    """
+    if "experiment_name" not in benchmarking_config and not all(
+        "experiment_name" in experiment for experiment in benchmarking_config["experiments"]
+    ):
+        raise ValueError("You must either specify a global experiment name or an experiment name for each experiment.")
+    if "export" not in benchmarking_config:
+        raise ValueError(
+            """You must specify export parameters. Example:
+            export:
+              export_artifacts: true
+              export_base_path: s3://benchmarking.us-west-2.ludwig.com/bench/    # include the slash at the end.
+        """
+        )
+    if "experiments" not in benchmarking_config:
+        raise ValueError("You must specify a list of experiments.")
```

Comment:
nit: Move these three and the raise on 191 to a separate `validate_benchmarking_config()` function.

This will require two traversals through the benchmarking_config (one in validation and one in propagation), but the separation of validation and propagation is clearer.